mobile_app_connector/models/account_invoice.py

```diff
@@ -119,50 +62,55 @@ def mobile_post_invoice(self, json_data, **parameters):
             invoice_lines_values.append(l_vals)
```

Comment:
With the 50.- sponsorships, the staff will have to split the invoice.lines such that the 8CHF end in the general fund.
(In the cases where the new invoice does not match any existing open invoice)

Reply:
Yes, I think we can live with that.